### PR TITLE
Added some missing members to googlemaps opening_hours

### PR DESCRIPTION
**Pull request by amritk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developers.google.com/maps/documentation/javascript/reference>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.(n/a)

### CODE REVIEW

**Comment by amritk** [comment]
where is @dt-bot

**Comment by amritk** [comment]
@mhegazy any idea why I didn't get any @dt-bot love in this one?

**Comment by uniqueiniquity** [comment]
@cgwrench, @nertzy, @xaolas, @mrmcnerd, @martincostello, could you review this PR?

File: types/googlemaps/index.d.ts [hunk 1]
```diff
@@ -2476,19 +2476,22 @@ declare namespace google.maps {
...
+            close?: OpeningHoursTime;
         }
 
         export interface OpeningHoursTime {
-            day: number,
-            time: string
+            day: number;
+            hours: number;
+            minutes: number;
+            nextDate: number;
```
Comment (martincostello) [inline types/googlemaps/index.d.ts]: Is `nextDate` correct as a `number`?

Comment (amritk) [reply types/googlemaps/index.d.ts]: @martincostello it is in fact a number

Comment (martincostello) [reply types/googlemaps/index.d.ts]: Ok cool. Wasn’t obvious from their naming it’d be an epoch.

**Comment by amritk** [comment]
Good to go then?

**Comment by martincostello** [comment]
:+1

**Pull request merged**